hazelcast-sql/src/test/java/com/hazelcast/jet/sql_nightly/SqlSTSInnerNonEquiJoinFaultToleranceStressTest.java

```diff
@@ -19,14 +19,19 @@
 import com.hazelcast.test.HazelcastParametrizedRunner;
 import com.hazelcast.test.HazelcastSerialParametersRunnerFactory;
 import com.hazelcast.test.annotation.NightlyTest;
+import com.hazelcast.test.annotation.ParallelJVMTest;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
 @RunWith(HazelcastParametrizedRunner.class)
 @Parameterized.UseParametersRunnerFactory(HazelcastSerialParametersRunnerFactory.class)
-@Category(NightlyTest.class)
+@Category({NightlyTest.class, ParallelJVMTest.class})
 public class SqlSTSInnerNonEquiJoinFaultToleranceStressTest extends SqlSTSInnerEquiJoinFaultToleranceStressTest {
+    static {
+        SINK_COUNT = 200;
```

Comment:
depending on the order of test execution you can run `SqlSTSInnerEquiJoinFaultToleranceStressTest` with changed config if SqlSTSInnerNonEquiJoinFaultToleranceStressTest is executed first in the same JVM. Do these fields have to be static?